docs/source/quickstart.mdx

```diff
@@ -0,0 +1,545 @@
+Setting up [products] subgraph at http://localhost:4001/products/graphql
+Setting up [orders] subgraph at http://localhost:4001/orders/graphql
+Setting up [users] subgraph at http://localhost:4001/users/graphql
+...
+```
+
+Keep these subgraphs up and running so that the router can query them.
+
+<Note>
+
+The retail demo supergraph contains the code behind each of these subgraphs, including their schemas, resolvers, and mock data.
+These files are provided for your reference—understanding them isn't necessary for completing this quickstart. Check out these tutorials to learn more about [GraphQL schemas](https://www.apollographql.com/tutorials/lift-off-part1) and [resolvers](https://www.apollographql.com/tutorials/lift-off-part2).
+</Note>
+
+#### Run router
+
+With your subgraphs running, you can use Rover to start the router locally.
+To do so, use the `rover dev` command with the `--supergraph-config` and `--router-config` options.
+The demo retail supergraph comes with `supergraph-config-dev.yaml` and `router-config-dev.yaml` YAML configuration files that include subgraph names, URLs, and router configurations.
+
+In a different terminal window from the one where the subgraphs are running, run this command:
+
+```bash showLineNumbers=false
+rover dev \
+  --supergraph-config supergraph-config-dev.yaml \
+  --router-config router-config-dev.yaml
+```
+
+When you first start the `rover dev` process, you'll see the following happening in the terminal:
+
+- Rover obtains the provided subgraph schemas and starts sessions for each of them.
+
+  ```bash showLineNumbers=false
+  🛫 starting a session with the 'discovery' subgraph
+  🛫 starting a session with the 'checkout' subgraph
+  🛫 starting a session with the 'users' subgraph
+  🛫 starting a session with the 'inventory' subgraph
+  🛫 starting a session with the 'orders' subgraph
+  🛫 starting a session with the 'shipping' subgraph
+  🛫 starting a session with the 'products' subgraph
+  🛫 starting a session with the 'reviews' subgraph
+  ```
+
+- Rover uses Apollo Federation to compose a supergraph schema by adding subgraph schemas one at a time.
+
+  ```bash showLineNumbers=false
+  🎶 composing supergraph with Federation v2.8.0
+  ✅ successfully composed after adding the 'products' subgraph
+  ```
+
+  - The order in which Rover adds subgraphs is undefined, so you might see composition errors like the following during intermediate steps.
+
+  ```bash showLineNumbers=false
+  error[E029]: Encountered 1 build error while trying to build a supergraph.
+  ```
+
+- Once composition completes successfully, Rover starts a locally running router session and provides it with the supergraph schema.
+
+  ```bash showLineNumbers=false
+  🚀 your supergraph is running! head to http://localhost:4000 to query your supergraph
+  ```
+
+- Rover starts watching the provided subgraph schemas for changes.
+
+  ```bash showLineNumbers=false
+  👀 watching ./subgraphs/products/schema.graphql for changes
```

Comment:
getting some weird highlighted words. 

<img width="773" alt="Screenshot 2024-06-21 at 16 07 19" src="https://github.com/apollographql/federation/assets/1368445/a7ea368c-2567-4afb-8710-52d12014298e">

Reply:
Seems to be a side effect of using `bash` or `shell`. Using `sh` alleviates this, but not sure if it's the most accurate

<img width="823" alt="image" src="https://github.com/apollographql/federation/assets/23219998/b4c85bc3-507e-46ad-a2cb-711e6283b79b">

Reply:
https://github.com/apollographql/federation/pull/3037/commits/5fc4f6e6c127cc23544f1452ab5c0381b6c865da